### ISSUE TITLE
Feature Request - Terminal Logging/Export

### ISSUE THREAD

**Issue by hbland-cjc** [GoogleCloudPlatform/iap-desktop#1145]
Hi,

Hope this okay.

It would be great to have a feature to Log Terminal Sessions to file or have an export feature to export the Terminal History.

Thanks :)

**Comment by jpassing** [comment]
Is there a specific use case that you have in mind? 

Would you expect the output to be the raw xterm stream (which can look quite messy if you used interactive applications like `vim` or `top`), or would you expect it to be in a format that can be played back using [asciinema.org](https://asciinema.org/) (or something similar)?

**Comment by hbland-cjc** [comment]
Yes, normally I will log sessions to use in documentation similar to asciicinema or if debugging an application issue a history of commands and response can help to review post debug.

Format ideally would be just raw from the terminal however could see the benefit of having the option for both, in different scenarios maybe a verbosity filter to reduce or increase the level of log saved to file?

I've included below a couple examples from programs I've previously utilised this type of feature which might be able to explain better than myself what I mean.

https://codingpackets.com/blog/securecrt-logging/

https://docs.devolutions.net/kb/remote-desktop-manager/how-to-articles/send-ssh-logs-verbose/

https://my.kualo.com/knowledgebase/?kbcat=0&article=888

**Comment by jpassing** [comment]
Thanks. I agree that this would be a good feature to add.
